src/components/modals/ScanModal.tsx

```diff
@@ -36,10 +38,14 @@ interface Props {
   textModalBody?: React.ReactNode | string
   textModalHint?: string
   textModalTitle?: string
+
+  /** Display the asset icon */
+  wallet?: EdgeCurrencyWallet
+  tokenId?: EdgeTokenId
```

Comment:
This whole change is a misunderstanding of the task. The task is to `Put asset icon and subicon on top of QRcode` not on top of the scanning camera view. This should go on to the request scene